DP Mystery Map/Assets/Scripts/PlayerController.cs

```diff
@@ -137,12 +146,17 @@ void Start()
             { Direction.Left, Player.MoveLeftKey},
             { Direction.Right, Player.MoveRightKey }
         };
+
+        Player.FacingDirection = Direction.Down;
```

Comment:
Remove this. This forces the player to face downwards even though save files save the player's facing direction. If this is for the animator, then have the animator itself change in `Start()`